jcl/src/java.base/share/classes/java/lang/Access.java

```diff
@@ -727,6 +736,15 @@ public String getLoaderNameID(ClassLoader loader) {
 	}
 /*[ENDIF] JAVA_SPEC_VERSION >= 11 */
 
+/*[IF JAVA_SPEC_VERSION >= 22]*/
+	public boolean bytesCompatible(String str, Charset charset) {
+		return str.bytesCompatible(charset);
+	}
+	public void copyToSegmentRaw(String str, MemorySegment ms, long offset) {
+		str.copyToSegmentRaw(ms, offset);
+	}
```

Comment:
Please add a blank line to separate these new methods.
Each should have the `@Override` annotation.
Please use less cryptic parameter names (suggest `string` and `segment`).

Reply:
Updated